web/Controllers/LandlordController.cs

```diff
@@ -141,4 +141,28 @@ public IActionResult ViewProperties()
         var listOfProperties = databaseResult.Select(PropertyViewModel.FromDbModel).ToList();
         return View("Properties", new PropertiesDashboardViewModel(listOfProperties));
     }
+    
+    [HttpGet]
+    [Route("edit")]
+    public ActionResult EditProfilePage(LandlordProfileModel currentProfileModel)
+    {
+        return View("EditProfilePage", currentProfileModel);
+    }

```

Comment:
This relies on the `LandlordProfileModel` already being populated in the current session. Try restarting the server and then going straight to `/landlord/edit`. We should load the landlord record from the database in this method.

It would also be good to follow the same URL convention that the profile uses, i.e. `/landlord/id:int/profile` and `/landlord/me/profile` - that makes it easy for us to extend this to admins that don't have a specific landlord associated with them. That's a bit more work though, so probably better to leave for BNH-45

Reply:
Just a couple of quick clarifications: 
What do you mean by restarting the server? Do you just mean get rid of the current page or is there some kind of method I should use?
Also would you like me to do both of these changes in BNH-45 or just the URL changes in BNH-45
Thanks :)

Reply:
Press stop and start in Rider.

Just the URL changes, should make the change to load the landlord in this method now